packages/amplify-graphql-directives/src/directives/v1/connection.ts

```diff
@@ -0,0 +1,20 @@
+import { Directive } from '../directive';
+
+const name = 'connection';
+const definition = /* GraphQL */ `
+  directive @${name}(
+    name: String
+    keyField: String
+    sortField: String
+    keyName: String
+    limit: Int
+    fields: [String!]
+  ) on FIELD_DEFINITION
+`;
+const defaults = {};
+
+export const ConnectionDirective: Directive = {
```

Comment:
I'm a bit torn, but I wonder if we should just go ahead & name this `ConnectionDirectiveV1`? I realize we don't technically need to disambiguate it, but having the built-in marker for these legacy directives is kind of nice. Thoughts?

Reply:
Thinking about this now again I think `ConnectionDirectiveV1` would be better. There isn't really a drawback to using the `V1` suffix.